files/lovely-icons/update.json

```diff
@@ -3,6 +3,6 @@
   "name": "lovely-icons",
   "repo": "douglasdeodato/lovely-icons",
   "files": {
-    "include": ["./css/lovely-icons.min.css", "./lovely-icons/*"]
+    "include": ["./css/lovely-icons.min.css", "./lovely-icons/lovely-icons.min.css", "./lovely-icons/*"]
```

Comment:
@MartinKolarik  and @megawac  I just put back the folder and release a new version, so now the bot should pick this version? thanks for the help.